packages/rum/src/domain/record/utils.ts

```diff
@@ -28,3 +29,64 @@ export function assembleIncrementalSnapshot<Data extends BrowserIncrementalData>
     timestamp: timeStampNow(),
   }
 }
+
+export function checkStyleSheetAndCallback(styleSheet: CSSStyleSheet | null, callback: (id: number) => void): void {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏move this function in `initStyleSheetObserver` as it is rather specific to its implementation, and there is no tests associated